supported-services.json

```diff
@@ -3,6 +3,24 @@
         "php": [
             8.1,
             8.2
+        ],
+        "composer": [
+            2.2
+        ],
+        "database": [
+            "mysql:8.0"
+        ],
+        "search": [
+          "elasticsearch:7.17"
+        ],
+        "message_queue": [
+            null

```

Comment:
Since warden spins up a rabbitmq container, why not use it?

Reply:
We actually can put here quite a big map services and all of them will be taken into account by matrix. The thing is Rabbit MQ is actually not needed for integration tests - so I just left dummy value of null. Not a big deal to change it.  

Actually, this is the next questions that we need to address 
- `Which services and versions should we count as the most valuable for testing needs?` 
I assume this should be the full list of supported services&versions of Magento Open Source based on release notes.  
- `How do we handle this? Should we use one list of support services for all tests? Would take too much run-time in pipelines?`  

Thx @Vinai

Reply:
Follow up question&suggestion:

`Do we need setup tests? To make sure Mage-OS can be setup and run on various services&versions`. I think we can actually just run integration&unit tests on one most common pre-set, but run setup tests before release on all variations.

Reply:
I believe there are integration tests that test the AMQP integration. If there is no queue configured, these tests are skipped. So, in order to run the full test suite, we need rabbitMQ.

For example `\Magento\AsynchronousOperations\Model\MassScheduleTest`.

Reply:
> Which services and versions should we count as the most valuable for testing needs?

Ideally this should be an optional input, to make it possible to run only a subset of service versions, or by default all supported versions.

Reply:
> How do we handle this? Should we use one list of support services for all tests? Would take too much run-time in pipelines?

The mid-term goal is to use https://github.com/mage-os/github-actions/tree/main/supported-version
However, this will require adding all supported versions to the returned matrix of that action.